Lib/test/test_clinic.py

```diff
@@ -3294,6 +3294,7 @@ def test_meth_method_no_params(self):
         check = partial(self.assertRaisesRegex, TypeError, "no arguments")
         check(meth, 1)
         check(meth, a=1)
+        obj.test_meth_method_no_params_misuse()
```

Comment:
This fails if I revert the clinic.py part of 355accc58ab767e55735cbf25417221020c7ec2f:

```
======================================================================
ERROR: test_meth_method_no_params (test.test_clinic.ClinicFunctionalTest.test_meth_method_no_params)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/erlend.aasland/src/python/main/Lib/test/test_clinic.py", line 3297, in test_meth_method_no_params
    obj.test_meth_method_no_params_misuse()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
TypeError: meth_method_no_params() takes no arguments

```